curriculum/templates/curriculum/lessonplan_find_share.html

```diff
@@ -0,0 +1,147 @@
+                <div class="col mb-3">
+                    <div class="btn-group btn-group-toggle" data-toggle="buttons">
+                        <label class="btn btn-outline-primary {% if 'ES' in form.grade_level.value %}active{% endif %}">
+                            <input type="checkbox" name="grade_level" value="ES" autocomplete="off" {% if 'ES' in form.grade_level.value %}checked{% endif %} /> Elementary School
+                        </label>
+                        <label class="btn btn-outline-primary {% if 'MS' in form.grade_level.value %}active{% endif %}">
+                            <input type="checkbox" name="grade_level" value="MS" autocomplete="off" {% if 'MS' in form.grade_level.value %}checked{% endif %} /> Middle School
+                        </label>
+                        <label class="btn btn-outline-primary {% if 'HS' in form.grade_level.value %}active{% endif %}">
+                            <input type="checkbox" name="grade_level" value="HS" autocomplete="off" {% if 'HS' in form.grade_level.value %}checked{% endif %} /> High School
+                        </label>
+                        <label class="btn btn-outline-primary {% if 'U' in form.grade_level.value %}active{% endif %}">
+                            <input type="checkbox" name="grade_level" value="U" autocomplete="off" {% if 'U' in form.grade_level.value %}checked{% endif %} /> Post-secondary
+                        </label>
+                    </div>
+                </div>
+                <div class="col mb-3">
+                    <div class="btn-group-toggle" data-toggle="buttons">
+                        <label class="btn btn-outline-primary {% if form.web_only.value %}active{% endif %}" for="{{ form.web_only.id_for_label }}">
+                            <input type="checkbox" autocomplete="off" name="{{ form.web_only.name }}" {% if form.web_only.value %}checked{% endif %} /><i class="fab fa-chrome"></i> {{ form.web_only.label }}
+                        </label>
+                    </div>
+                </div>
+                <input type="hidden" name="user_id" value="{% if form.user_id.value %}{{ form.user_id.value }}{% endif %}" />
+            </div>
+        </form>
+        <div class="row my-5">
+            {% for lessonplan in object_list %}
+                <div class="col-4">
+                    <div class="curriculum mb-3">
+                        <img class = "img-responsive mb-3" 
+                            src="{% static 'img/computer_heart.jpg'%}" 
+                            alt="Computer chip heart"
+                            height="auto"
+                            width="100%"
+                        >
+                        <h6 class="flex-fill curriculum-title text-primary">{{ lessonplan.title }}</h6>
+                        {% if lessonplan.web_only %}
+                            <p class="pr-4" title="Chromebook compatible"><i class="text-right fab fa-chrome"></i></p>
```

Comment:
Could we maybe put the star rating and chromebook symbol on one line? I don't feel like it deserves its own line